packages/pcd/eddsa-pcd/test/EDDSAPCD.spec.ts

```diff
@@ -1,7 +1,14 @@
 import { ArgumentTypeName } from "@pcd/pcd-types";
```

Comment:
When this is finalized, I suggest this package (and-or the library packages which fork off of it) should have its own compatibility tests similar to those in @pcd/pod.  In particular compatibility between circomlibjs and zk-kit, and backward-compatibility with old EdDSA PCD formats are both worth covering.  I think I already made some basic end-to-end compatibility tests based on serialized PCDs in strings, but something more directed and detailed would probably be good.